### PR TITLE
Added support for django 1.11

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**
LEARNER-1182

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
     license='AGPL',
     url='http://github.com/edx/edx-django-release-util',
     install_requires=[
-        'django>=1.8,<1.11',
+        'django>=1.8,<2.0',
```
Comment (schenedx) [inline setup.py]: If we do this, and Django decided to have django 1.12, we will be suffering from breakage.
Are you sure Django won't have version 1.12? Why not update this to be `<1.12`?

File: tox.ini [hunk 2]
```diff
@@ -10,6 +10,7 @@ deps =
     django18: Django>=1.8,<1.9
     django19: Django>=1.9,<1.10
     django110: Django>=1.10,<1.11
+    django111: Django>=1.11,<2.0
```
Comment (schenedx) [inline tox.ini]: Same comment as above. Why not do `django111: Django>=1.11,<1.12`?

File: setup.py (hunk 1 above)
Comment (clintonb) [inline setup.py]: https://www.djangoproject.com/weblog/2015/jun/25/roadmap/

**Comment by clintonb** [comment]
Update your commit message: Added support for Django 1.11 and Python 3.6

File: setup.py (hunk 1 above)
Comment (schenedx) [inline setup.py]: @clintonb I got it. Thanks.

**Pull request merged**